src/main/java/net/sargue/time/jsptags/FormatTag.java

```diff
@@ -29,72 +30,73 @@
  * @author Jim Newsham
  * @author Sergi Baila
  */
-@SuppressWarnings("UnusedDeclaration")
 public class FormatTag extends FormatSupport {
 
-    /**
-     * Sets the value attribute.
-     *
-     * @param value  the value
-     */
-    public void setValue(Object value) {
-        this.value = value;
-    }
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * Sets the value attribute.
+	 *
+	 * @param value the value
+	 */
+	public void setValue(final Object value) {
```

Comment:
Why the final modifier on the parameter?

Reply:
Good coding practice. Changing the value of parameters inside a function makes reading the code hard. In general I like to make everything final that can be so that when reading code it's clear what can change and what can't. It's also a minor optimization for the compiler.

Reply:
Well, I prefer to keep the changes as minimum required to the point of the change itself. Nor this is the place to discuss about code style or the performance. I also value consistency a lot, so changing a couple of instances doesn't work for me.

If you think marking the parameters are final is worth it due to performance then let's discuss it in another PR where all the code is refactored.

Reply:
@sargue I understand the desire to keep the changes to a minimum. I've updated the PR to minimize changes. The largest changes are in FormatTagTest.java. I found that the tests needed to be modified to pass. I suspect that the Locale formats have changed since you originally wrote it. I also ran into issues with the definition of the first day of the week. I worked to try and keep the same tests and make them robust to such changes so that they can be run on any system.

Reply:
Thanks, I appreciate it.

The idea of the locale testing was probably a bad approach to begin with, so I really appreciate you working to keep them. This almost look final.